apix-impl/build.gradle

```diff
@@ -27,11 +45,11 @@ subprojects {
     artifacts {
         ampArtifact amp
     }
-
+    
     sourceSets {
         main {
             java {
-                srcDirs = ['src/main/java', 'src/main/java-shared']
+                srcDirs = ['src/main/java', '../src/main/java']
```

Comment:
@WimCrols I think it is more clear to change `'00/src/main/java'` to `"${project(":apix-integrationtests").path}/src/main/java"`. What do you think?

Reply:
I considered it but think the ../ is actually more readable.

Reply:
That is also ok for me. It is probably a matter of opinion. I will approve the changes.